build/configs/artik053/fault-manager/Make.defs

```diff
@@ -0,0 +1,173 @@
+###########################################################################
+#
+# Copyright 2019 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+############################################################################
+# build/configs/artik053/fault-manager/Make.defs
```

Comment:
Is it taken from Nuttx? If not, license is not required.

Reply:
I think every Make.defs present in configs/artik053 folder follows the same licensing.
What needs to be done?

Reply:
Just remove the nnuttx license.